Fix broken call of sourceCode

ADimeo opened the pull request:
This fixes a broken (on my system) method call.

The method CompiledMethod sourceCode does not exist on my system. CompiledMethod getSource does, however. I do not know how the former call ended up in our image, since none of the SWE-images include this call.

ADimeo committed:
Fix broken call of sourceCode

arminWel commented:
The method `#getSource:` is not implemented(at least not in my image), while `#sourceCode:` is. Therefore is must recline this PR.  Reasons for the differences in the images should be discussed with the chair as soon as possible(maybe monday morning? @ADimeo).

arminWel commented:
For some reason (those must be quite weird), my Image know has the method the way you have. We should definitely ask at the chair about that-

ADimeo committed:
Fix incompatible tests

Some tests did not run on jenkins.
	- 'Signals' class is not core to the image, and was missing
	- Aborted tests led to source code being changed permanently, which inhibited checking for differences
	- self deny: equals: is not a stable method yet, and missing in older squeak builds

arminWel committed:
Ensure changes after rebase

arminWel commented:
There are problems with older versions of squeak. Thus have to be solved, before we merge. I'm working on it.

arminWel committed:
reset the Session after running a test on it

arminWel committed:
Use methods is the tests, which are supported since 5.1

Prior to this change, test would fail on older versions of squeak

arminWel committed:
Use a dummy class instead of an external one for the test to manipulate

arminWel committed:
deletes Squeak 5.0 from the travis.yml

There is a bug in Squeak 5.0, which prevents our build from being
successfull. We got the permission by our tutor to delete the dependency
for now.

PaulBrachmann commented on packages/CollabCodeEdit-Tests.package/CCEModelTest.class/instance/tearDown.st:
```diff
@@ -1,4 +1,5 @@
 	model changed: #close.
-	self newMorphs do: [:morph | morph model changed: #close]
\ No newline at end of file
+	self newMorphs do: [:morph | morph model changed: #close].
```
```suggestion
	self newMorphs do: [ :morph | morph model changed: #close ].
```

PaulBrachmann commented on packages/CollabCodeEdit-Tests.package/CCEPackageManagerTests.class/instance/testHashchangesAfterMetaClassInformationChange.st:
```diff
@@ -2,18 +2,13 @@ tests
 	originalComment := testClass comment.
- 
-	hashBeforeChange := packageManager hashPackages: #( 'Signals' ).
+	[ hashBeforeChange := packageManager hashPackages: #( 'CollabCodeEdit-Tests' ).
```
Code following a multiline block being opened should be indented. (See lines below.)

PaulBrachmann commented on packages/CollabCodeEdit-Tests.package/CCEPackageManagerTests.class/instance/testHashchangesAfterMetaClassInformationChange.st:
```diff
@@ -2,18 +2,13 @@ tests
-	testClass comment: originalComment
\ No newline at end of file
+	self deny: hashBeforeChange = hashAfterChange. ]
+	ensure: [testClass comment: originalComment]
```
```suggestion
	ensure: [ testClass comment: originalComment ]
```

PaulBrachmann replied:
The current indentation makes it unclear that `ensure:` belongs to the preceding block.

PaulBrachmann commented on packages/CollabCodeEdit-Tests.package/CCEPackageManagerTests.class/instance/testHashchangesAfterMetaClassInformationChange.st:
```diff
@@ -2,18 +2,13 @@ tests
-	
-	testClass comment: originalComment
\ No newline at end of file
+	self deny: hashBeforeChange = hashAfterChange. ]
```
```suggestion
	self deny: hashBeforeChange = hashAfterChange ]
```

PaulBrachmann reviewed (changes requested):
Looks great. It is indeed quite the relief to finally see a build passing!

There are however some minor style inconsistencies that should be fixed.
For further information, refer to our [Smalltalk Style Guide](https://docs.google.com/document/d/11PfOTymUnovjca5ExAI6-6-cC16f6q5mQ8juo9Wn13I/edit?usp=sharing).

arminWel committed:
Update packages/CollabCodeEdit-Tests.package/CCEModelTest.class/instance/tearDown.st

Co-Authored-By: Paul Brachmann <paul@enormos.com>

arminWel committed:
Update packages/CollabCodeEdit-Tests.package/CCEPackageManagerTests.class/instance/testHashchangesAfterMetaClassInformationChange.st

Co-Authored-By: Paul Brachmann <paul@enormos.com>

arminWel committed:
Update packages/CollabCodeEdit-Tests.package/CCEPackageManagerTests.class/instance/testHashchangesAfterMetaClassInformationChange.st

Co-Authored-By: Paul Brachmann <paul@enormos.com>

arminWel committed:
resolve requested changes
